source/solvers/navier_stokes_scratch_data.cc

```diff
@@ -111,6 +111,40 @@ NavierStokesScratchData<dim>::enable_VOF(const FiniteElement<dim> &fe,
   phase_gradient_values = std::vector<Tensor<1, dim>>(this->n_q_points);
 }
 
+template <int dim>
+void
+NavierStokesScratchData<dim>::enable_filtered_phase_fraction_gradient(
+  const FiniteElement<dim> &fe_filtered_phase_fraction_gradient,
+  const Quadrature<dim> &   quadrature,
+  const Mapping<dim> &      mapping)
+{
+  gather_filtered_phase_fraction_gradient = true;
+  fe_values_filtered_phase_fraction_gradient =
+    std::make_shared<FEValues<dim>>(mapping,
+                                    fe_filtered_phase_fraction_gradient,
+                                    quadrature,
+                                    update_values | update_gradients);
+
+  // PFG
```

Comment:
Phase fraction gradient? Write it more clearly here please

Reply:
done.